provider/app.py

```diff
@@ -65,6 +66,13 @@ def testRoute():
     return jsonify('pong')
 
 
+@app.route('/health')
+def healthRoute():
+    currentTime = datetime.now()
+    diff = currentTime - startTime
+    return jsonify({'uptime': diff.seconds})
```

Comment:
Yup, agreed. There are more stats we could include that are documented in #7. For now I just want to get the uptime implemented to ensure we understand how Docker maintains state over time.

I'll switch the `uptime` to include days, hours, minutes, and seconds.